{{cookiecutter.project_slug}}/Makefile

```diff
@@ -46,6 +46,12 @@ update: ## Pull the latest image from a registry
 	@docker pull node:16-bullseye-slim
 	@docker pull redis:6.2
 
+1pass:  ## Create a vault and items in 1Password
+	op vault create "{{ cookiecutter.project_name }}"; \
```

Comment:
Without getting into the details, our current policies and procedures around 1Password vault management are a bit more complex than this, and this may contradict some of those.  Do we need to adjust policies?

Reply:
Not sure what exactly you have in mind ... but I can adjust the code ...

The snippet above is to serve only as a bootstrap to get you up to speed asap.
I am working a bit closer now with the scaf project and what I had to do is to create all that manually - pretty tedious.

Let me know what policies we should apply here.

Reply:
A generic solution is probably fine for Scaf, but we don't want to add short-term convenience commands to Scaf. We should put our efforts into moving to an external secrets operator and automate the bootstrapping of secrets, not run make commands manually.

Reply:
Do we keep the PR and fix it, or just close it for now?

Reply:
Just close it for now.